tests/phpunit/class-wc-stripe-webhook-handler-test.php

```
@@ -849,6 +849,123 @@ public function test_process_webhook_refund_updated( $notification_status, $emai
+		};
+		add_action( 'wc_gateway_stripe_process_webhook_payment_error', $hook, 10, 2 );
+
+		$this->mock_webhook_handler->process_checkout_session_failure( $notification );
+		remove_action( 'wc_gateway_stripe_process_webhook_payment_error', $hook, 10 );
+
+		$resolved_order_property = new ReflectionProperty( WC_Stripe_Webhook_Handler::class, 'resolved_order' );
+		$resolved_order_property->setAccessible( true );
+
+		$this->assertNull( $resolved_order_property->getValue( $this->mock_webhook_handler ) );
+		$this->assertSame( 0, $hook_calls );
+	}
```

PHPMD (2.15.0)

[error] The method test_process_checkout_session_failure_returns_when_order_is_not_found is not named in camelCase. (undefined)

(CamelCaseMethodName)